### ISSUE TITLE
Ghosts can't properly examine paper attached to an airlock

### ISSUE THREAD

**Issue by RedFoxIV** [tgstation/tgstation#40069]
![image](https://user-images.githubusercontent.com/38788538/44996557-35b61e00-afb1-11e8-83c1-5cc91fdfc3ae.png)
![image](https://user-images.githubusercontent.com/38788538/44996563-3b136880-afb1-11e8-9a34-9a3ec3be8525.png)

**Comment by AlexTheSysop42** [comment]
Did you try it when next to the airlock

**Comment by octareenroon91** [comment]
># Did you try it when next to the airlock

**Comment by Arkvold** [comment]
Yeah, this is a case of user-error, unless ghosts are supposed to be able to examine paper on airlocks at a distance.  It's working just fine for me.

![image](https://user-images.githubusercontent.com/27884105/45071775-eb629900-b0a5-11e8-96a3-3dba8b9f135c.png)
